codeforboston/vaccinatema
# Introduce next.js for migration to React.

@snyderc, @mwellman17, and I are hoping to migrate the frontend to React! Chris found that [Next.js](https://nextjs.org) will allow us to serve React pages alongside our existing EJS pages and Express endpoints. Next also allows for server-side rendering for better SEO and has internationalization support that we might be able to use.

This first change adds the necessary React and Next dependencies and adds a temporary React page as a proof of concept. Going forward, we'll actually start migrating to React!

### Test Plan:
* Ensure the whole website functions as usual (with no console errors).
* Double check that the search functionality works as expected.
* Go to localhost:3002/dev and see "Hello, world!".

Co-authored-by: Chris Snyder <christopher.snyder.m@gmail.com>
